src/sampler.jl

```diff
@@ -8,14 +8,19 @@ function reconstruct(
     return reconstruct(h, metric=metric)
 end
 
-reconstruct(τ::AbstractProposal, ::AbstractAdaptor) = τ
+reconstruct(κ::AbstractProposal, ::AbstractAdaptor) = κ
 function reconstruct(
-    τ::AbstractProposal, adaptor::Union{StepSizeAdaptor, NaiveHMCAdaptor, StanHMCAdaptor}
+    κ::AbstractProposal, adaptor::Union{StepSizeAdaptor, NaiveHMCAdaptor, StanHMCAdaptor}
 )
+    return reconstruct(κ, getϵ(adaptor))
+end
+function reconstruct(κ::AbstractProposal, ϵ::AbstractScalarOrVec{<:Real})
```

Comment:
Is this new `reconstruct` function used elsewhere? If not, maybe we can keep the old implementation which avoids creating one additional function.

Reply:
It's used in the test.

Reply:
In that case, I suggest we remove this function.

Reply:
Sounds good. I will merge it back to Line 12.